modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -51,11 +51,12 @@
 			fsmState: [],
 			screenShown: false,
 			currentScreenNameShown: "",
-
+			externalControlManager : null,
 			setup: function ( ) {
 				this.initConfig();
 				this.initDisplays();
 				this.initFSM();
+				this.initExternalControlManager();
```

Comment:
@OrenMe I moved the initialization into the setup so it won't be affected from the changing media process, please confirm...